loadtest/control/simplecontroller/actions.go

```diff
@@ -129,6 +129,20 @@ func (c *SimpleController) viewChannel() control.UserStatus {
 	*/
 }
 
+func (c *SimpleController) patchUser(username, nickname, firstName, lastName *string) control.UserStatus {
```

Comment:
We are trying to keep actions parameterless when possible. The reason is we want to keep the decision/complexity of what to update inside and not just wrap once again the underlying api call. Also I think this should be named `updateProfile`. The action's name should reflect what the user does at a higher level.